bin/propolis-server/src/lib/server.rs

```diff
@@ -105,7 +107,7 @@ pub enum VmControllerState {
     Destroyed {
         /// A copy of the instance properties recorded at the time the instance
         /// was destroyed, used to serve subsequent `instance_get` requests.
-        last_instance: api::Instance,
+        last_instance: Box<api::Instance>,
```

Comment:
Why did this get Boxed?  Did the compiler complain about size?

Reply:
Yeah, there was a clippy lint about the size of the enum in CI. I'm not sure why I hit it now.

Reply:
For reference: https://github.com/oxidecomputer/propolis/actions/runs/8197097085/job/22418514347#step:5:1753